UnitTests/BraintreeSEPADirectDebitTests/BTSEPADirectDebitClient_Tests.swift

```diff
@@ -492,4 +492,90 @@ class BTSEPADirectDebitClient_Tests: XCTestCase {
             }
         }
     }
+
+    func testTokenize_withSEPADirectDebitRequest_returnsSEPADirectDebitNonce() async throws {
```

Comment:
Ah - switching all of the tests over is going to probably make the diff in this PR pretty huge, thoughts on either just updating a few tests for now instead of having new tests or removing these new tests and punting as a whole on updating the tests to use sync await?